napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -293,13 +293,48 @@ def _set_table(self, layer_str: str = ''):
             self._table.setItem(0, 0, item)
 
     def _get_layer_actions(self):
+        """
+        Get all the actions that could potencially raise a keybinding conflict
+        taking into account the current selected keybinding group.
+
+        Returns
+        -------
+        actions_all: tuple
+            Tuple with tuples of actions that can potencially enter in conflict
+            when changing keybindings.The returned tuple is formated in the
+            following way:
+                (
+                    ('keybinding_group_1', ('action_name', action)),
+                    ...
+                    ('keybinding_group_n', ('action_name_n', action_n)),
+                )
```

Comment:
nitpick, we have already described above that these are conflicting actions, so I think we don't need to go into as much detail here.
```suggestion
            Tuple of group names and actions, to avoid keybinding
            conflicts with.
            Format:
                (
                    ('keybinding_group', ('action_name', Action)),
                    ...
                )
```